### PR TITLE
port of BreakfastFinder to dotnet6

### PR DESCRIPTION
**Pull request by stephen-hawley**

### CODE REVIEW

File: dotnet/ios/ios12/BreakfastFinder/BreakfastFinder/ObjectDetector.cs [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+			case "confidenceThreshold":
+				return MLFeatureValue.Create (ConfidenceThreshold);
+			default:
+				return null;
+		}
+	}
+
+	public ObjectDetectorInput (CVPixelBuffer image, double iouThreshold, double confidenceThreshold)
+	{
+		Image = image ?? throw new ArgumentNullException (nameof (image));
```
Comment (tj-devel709) [inline dotnet/ios/ios12/BreakfastFinder/BreakfastFinder/ObjectDetector.cs:57]: OOh I like this!

**Review by tj-devel709 — APPROVED** [review]

File: dotnet/ios/ios12/BreakfastFinder/BreakfastFinder/VisionObjectRecognitionViewController.cs [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+                                Console.WriteLine ($"{error.LocalizedDescription}");
+                        }
+                }
+                catch (Exception x)
+                {
+                        Console.WriteLine (x.Message);
+                }
+                finally
+                {
+                        sampleBuffer?.Dispose ();
```
Comment (mandel-macaque) [inline dotnet/ios/ios12/BreakfastFinder/BreakfastFinder/VisionObjectRecognitionViewController.cs:111]: @rolfbjarne  is this correct?

File: dotnet/ios/ios12/BreakfastFinder/BreakfastFinder/VisionObjectRecognitionViewController.cs [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+                // start the capture
+                StartCaptureSession ();
+        }
+
+        void SetupLayers ()
+        {
+                detectionOverlay = new CALayer (); // container layer that has all the renderings of the observations
+                detectionOverlay.Name = "DetectionOverlay";
+                detectionOverlay.Bounds = new CGRect (0.0, 0.0, bufferSize.Width, bufferSize.Height);
+                detectionOverlay.Position = new CGPoint (rootLayer!.Bounds.GetMidX (), rootLayer.Bounds.GetMidY ());
```
Comment (mandel-macaque) [inline dotnet/ios/ios12/BreakfastFinder/BreakfastFinder/VisionObjectRecognitionViewController.cs:135]: Property inits

**Review by mandel-macaque — APPROVED** [review]
Approve, minos style issue, we should talk in the call if we are using the mono style I have noticed you and @tj-devel709  using diff styles.

**Pull request merged**